app_dart/cloudbuild_app_dart_deploy.yaml

```diff
@@ -17,7 +17,7 @@ steps:
           unverified_provenance.json
 
   # Verify provenance is valid before proceeding with deployment.
-  - name: 'golang:stretch'
+  - name: 'golang:latest'
```

Comment:
Do they have a stable version? Latest makes it break prone and not easy to reproduce.

Reply:
I've updated it to `golang:1.20`, the current latest version